tests/wpt/mozilla/tests/mozilla/interfaces.html

```diff
@@ -141,27 +141,19 @@
   "ImageData",
   "Image",
   "KeyboardEvent",
-  "Location",
-  "MediaError",
   "MediaList",
   "MediaQueryList",
   "MediaQueryListEvent",
   "MessageEvent",
-  "MimeType",
-  "MimeTypeArray",
   "MouseEvent",
   "NamedNodeMap",
   "Navigator",
   "Node",
   "NodeFilter",
   "NodeIterator",
   "NodeList",
-  "PageTransitionEvent",
   "Performance",
   "PerformanceTiming",
-  "Plugin",
-  "PluginArray",
-  "PopStateEvent",
```

Comment:
This change doesn't look right; I would expect `./mach test-wpt tests/wpt/mozilla/tests/mozilla/interfaces.html` to fail now, since these interfaces should still be exposed in the Window global.